src/Services/CereProc.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
+namespace ChrisJP\TTS\Services;
+
+use ChrisJP\TTS\Services\Service;
+use ChrisJP\TTS\Request;
+use ChrisJP\TTS\ReturnObjectTrait;
+
+/**
+ * CereProc
+ * 
+ * One of few services whose live demo allows you to change format and sample rate.
+ * Doesn't use their actual paid API, instead makes a request to their live demo PHP script
+ * mimicking coming from their live demo page.
+ */
+class CereProc implements Service
+{
+    use ReturnObjectTrait;
+
+    const baseURL = 'https://www.cereproc.com/themes/benchpress/livedemo.php';
+
+    const demoSite = 'https://www.cereproc.com/support/live_demo';
+
+    /**
+     * Full name of this service.
+     *
+     * @var string
+     */
+    private string $name = 'CereProc';
+
+    /**
+     * Short name of this service.
+     * Will be used in audio filenames so keep it short and no weird characters/spaces etc.
+     *
+     * @var string
+     */
+    private string $shortName = 'CereProc';
```

Comment:
Instead of repeating this, maybe remove it and check for the existence of `$shortName` and if it exists use it and if not just use `$name`